sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/models/NamespaceProperties.java

```diff
@@ -107,7 +106,7 @@ public OffsetDateTime getCreatedTime() {
      * @param createdTime the createdTime value to set.
      * @return the NamespaceProperties object itself.
      */
-    NamespaceProperties setCreatedTime(OffsetDateTime createdTime) {
+    public NamespaceProperties setCreatedTime(OffsetDateTime createdTime) {
```

Comment:
This was intentional. We didn't want people to be able to set the created time since this is done by the service.